### DISCUSSION TITLE
Upgrade version

### DISCUSSION THREAD

**Discussion by tXambe** [projectdiscovery/nuclei#5560] [Q&A] [answered]
Hello,

when I run nuclei shows me that my version is outdated but I have not been able to see how I can update the version.

¿ Anyone can say how ?

A greeting and thanks

**Comment by GeorginaReeder** [comment]
Hi @tXambe !

We have some documentation on this. Have you already given these methods a try? https://github.com/orgs/projectdiscovery/discussions/3679

Let us know if you're still having trouble after that.

**Comment by tXambe** [comment]
Hello,

I have tried this option and it has not updated

go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest 

then I tried pdtm and it didn't work either.

```
bin ./pdtm -i nuclei -u nuclei 

                ____          
     ____  ____/ / /_____ ___ 
    / __ \/ __  / __/ __ __  \
   / /_/ / /_/ / /_/ / / / / /
  / .___/\__,_/\__/_/ /_/ /_/ 
 /_/                         

                projectdiscovery.io

[INF] Current pdtm version v0.0.9 (latest)
[INF] nuclei: already installed
[INF] nuclei: already up to date
➜  bin nuclei -version

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   2.7.8

                projectdiscovery.io

[WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] Current Version: 2.7.8


nuclei -u https://altoromutual.com

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   2.7.8

                projectdiscovery.io

[WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[WRN] Found 12 templates with syntax error (use -validate flag for further examination)
[INF] Using Nuclei Engine 2.7.8 (outdated)
[INF] Using Nuclei Templates 9.4.2 (latest)



```

My system is :

NAME="Kali GNU/Linux"
VERSION_ID="2024.3"
VERSION="2024.3"
Linux K1 6.8.11-amd64

Thanks for the support

**Reply by dwisiswant0** [reply, marked as answer]
The latest major version is **v3**, not v2.

```bash
go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
```

**Reply by tXambe** [reply]
Hello,

sorry, but don't work , I have tested this on a number of different virtual machines with the same result.

A greeting and thanks

**Reply by dwisiswant0** [reply]
@tXambe - Are you sure you've executed the commands from my code snippet? Can you show me the outputs?

**Reply by tXambe** [reply]
Hello,

I run go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest and my version of go --> go version go1.23.0 linux/386

**Comment by GeorginaReeder** [comment]
Hi again! Thanks for this - we can take a closer look. @dwisiswant0 , is there anything that might be missed with the update here?
